packages/aws-cdk-lib/aws-rds/lib/serverless-cluster.ts

```diff
@@ -272,6 +272,19 @@ export interface ServerlessScalingOptions {
    * @default - automatic pause enabled after 5 minutes
    */
   readonly autoPause?: Duration;
+
+  /**
+   * The amount of time, in seconds, that Aurora Serverless v1 tries to find a scaling point to perform seamless scaling before enforcing the timeout action.
+   * Specify a value between 60 and 600 seconds.
+   */
+  readonly secondsBeforeTimeout? : Duration;
```

Comment:
Can we rename this `timeout`? Is there a reason for the user to need to know the timeout is passed to CFN as seconds? Especially considering the prop is a `Duration`.